roqoqo-qryd/src/tweezer_devices.rs

```diff
@@ -1299,7 +1302,22 @@ impl TweezerDevice {
                 // If no shifts are allowed by the device for this tweezer, then it's not valid
                 None => return false,
             }
+            // "Faking" the movement of the qubit
```

Comment:
Why are we faking the movement, instead of actually doing it?

Reply:
Reason explained above.